cpp/tests/filling/filling_tests.cu

```diff
@@ -35,13 +35,18 @@ TYPED_TEST_CASE(FillingTest, test_types);
 
 constexpr gdf_size_type column_size{1000};
 
+using column_validity_fn_t = std::function<bool(gdf_index_type)>;
```

Comment:
Delete this line and use `auto` on the next line. Simpler and clearer, since it's obvious that the variable stores a lambda.

```suggestion
```